packages/components/ng-ovh-payment-method/src/components/integration/choice/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,33 @@
+{
+  "ovh_payment_method_integration_choice_title": "Afin de confirmer votre demande, veuillez choisir votre moyen de paiement, et accepter les conditions d'utilisation.",
+  "ovh_payment_method_integration_choice_tag_recommended": "Conseillé",
+  "ovh_payment_method_integration_choice_payment_method_add_link": "[Rendez-vous sur la page <a href=\"#/billing/mean\">Moyens de paiement</a> pour en sélectionner un]",
```

Comment:
What's happen if we are not in dedicated manager ?

Reply:
Redirection link work actually, but if we're not in the manager... this need a bigger work to handle this urls